### PR TITLE
OCM-5648 | feat: support both yaml and json format for tuningconfigs

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/OCM-5648

### CODE REVIEW

File: cmd/create/externalauthprovider/externalauthprovider_suite_test.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-func TestIdp(t *testing.T) {
+func TestExternalAuthProvider(t *testing.T) {
```
Comment (chenz4027) [inline cmd/create/externalauthprovider/externalauthprovider_suite_test.go:26]: Not related, but I found that the test suite name is incorrect for external auth

File: cmd/create/tuningconfigs/spec.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        "name": "tuned-2-profile"
+      }
+    ],
+    "recommend": [
+      {
+        "priority": 20,
+        "profile": "tuned-2-profile"
+      }
+    ]
+ }
```
Comment (hunterkepley) [inline cmd/create/tuningconfigs/spec.json:14]: NP: lets add new lines at the end of this file, and the `spec.yaml` file

**Review by hunterkepley — APPROVED** [review]
LGTM, small nitpick

**Comment by philipwu08** [comment]
LGTM

File: cmd/create/tuningconfigs/spec.json (hunk 2 above)
Comment (chenz4027) [reply cmd/create/tuningconfigs/spec.json]: Thanks @hunterkepley , added new lines

**Pull request merged**